examples/nuclei_image_segmentation/Nuclei_Image_Segmentation.ipynb

```diff
@@ -0,0 +1,521 @@
Jbk9VKMuuxooUO08aWUkfXgOWZsFjD44OwHiyRwrYI8/OeUpXtIwxzJZWmzVSQYnZX6sC5+c0mNfC9bcFinTQ5QZM5ZE1mI+aOF0kQM18jUUlboRpdb4ktJNEve6QmytbqyWpsfB11uaaNrdpsb1uAmy2M/EvU5C5JpCXDMM9hX5CZxbEvR77cIiLjF4c/DGDfnusQwDvGch27xjHWcvreMSB5rB3Tc6aMZ2znU/Wfut6UTT3W52Pg9li/nwNGT/X7uWD4qcXBqbqew4qfqstBSGT0YBeRDewxbVPm1s+J2kY/30ETsLfN6e041OfpfHhp//49i4hwcXHBzc0NtdYxvayPjQMqYNQY+xa0n+9BXXGOekBQlHY4iHHAEp0JItsGO01qvAdxy/nQOMeUzB5l78yggxUHfhFARZeDCHIj2xlts6YpciNbH0F1/Jw5uwqMjgEOIiNz7P307wYfD6/L700MeIsBYRFYu2vM+fn5HuD0PsbPkS8wIsD2ur3fEZxHEA+7z2B0bfDz3QUk9iXqiyML7dfxdkYA6/1zsBylQJHdjdfy+RvZ5fjdEcFw1Fo/t7zQmsu2x2tqq+LGBtbUmFiBVBXR3OyTLK6+iLbsWJVBFYoBidQCyWqipTm17XbR1KLyW2BWw6wWRFYbpye0HfYgE2jHijNzBppTUlITOHaYjlUrLfmDAfPcArmSCHIGXRa+++vM5WXi9wqffhyQmsjnig4tuUOBn35cUYYZ/GPP/DqxOFMzYhWQPjE/h/NtYftgQM50vpahLGUhV0gUA7XA+VlCNTMUA+hslX6jlA0MDfxyk9hI4uatMZ2zc9vuN72noa3uTJktYC2Vzw93rFcDSTq4UW4+DvyvF4VvX3/D9atE7gfQbFnPwHbUS6K03K5aoMvNi/dM6FfwWFtWMKqxyVh64WEjrLewLdL0u7bm0BaMlUV2DLCk3byqWPBeVvtbBM0t2YRIywzXjYuoLEJXlVILmUQvbgFmKDV3FixYtLleNM1EaoBVWgIQNRQ7Xqcm0xmnalpaT/AhyLh407YDUV2H8YUW193N5/ORCYhbbuv1es/v0b844WmQeeiY6c8pkD0GnJ8q0y2vY8c8Ve8UtB6qO7KuzwWnhwD2c9nRfyuQfar8ufUcWoj42EWW1H9GBtZZH3/PZS3AyNL5g9l3CTyi2aPtoyG/PxDjQz4Cl0Pyg0N9+dKLiHBzc4OqjlHmPobeX9fDxh0VD+y6u7sbQaQvYodh4Pb2lvPz85GZe/v2LdfX13vMl9/TyAy7rtHvT9S0+j2JetUYwBQXKw5MI3s/ZfXcMmoa6OTXAPYCjaZBhpHljAx/ZJwduE3tpCLgjRIID6yLTgE+Ln4dJwm875HpHoZhzG7nIM7TPvu8jbsNPnY+zlFaEIMCvX3epul99PZGUO71Rz27+8H65yOypfHeeJuiNjaC3vid6gsldybwuenXjdp4/xc9krfb7V6WsJeUlweAseOjROw/ZYdP1KkrbQyXbwX79q2MVqEt6EtNv9i2eg1HGSiT2uyR2vHG7jG+1nLKNna4XV3sNc/+VHGAu2u/W0Jl1V1kveoI5FIStFPyWWKJcvGmsroRHj5tGTYdbIGirIcti1nH5pfC++uOeS183Qld7xmjDPR0i4QWy3pFi8aH5oJAInVC3zdNzhI2VUhbMVcEMb1un4AsDA3gyQa297DuK5shk7P1Xxrw6ueJ1686PlwNDLqhJFApqJq29+O7FXdbod9WrnoZb6zlkUioFGNkxRYX2rb6cxbmvQHGWmxcuwyabSJvq5IHGAallJZiOMxJbYNcW/89fa60uWCuATLOBZsYaZx0tQXsiQhdtgQGWnVvjkhzy3D3ApuVrqpu7HCTTzSydszcW7HLpbaTYMxw81BubRNbA33hvCzjF6qDBNhp1u7v71mtVuOXTgQE05X+qXIK0Hp9h44/1NZTDNup8w9d+xAj+1TdsR1PMb5P1RvrOgS0TzGoxxYAx86J75069iUlMntPsbr+e3TDiGDGH4B+TGQI/WHrda1WqxEQOFPkoCyySzGIbNre2L7pa19ycWZ1s9mMnrvAXiKFOJbA3kJiuVyOW7YOTAAuLy/He9L3PW/evBkXHH68X9/BhYMnXwTHextBnN+bqLF1WUQEXrG9Dga9ftdsRtDk4BYYg4mi5tLrjiDWZQ6RUYxzIO4UeBuAPUDuY+H99uDF6LoQ56YXP88BsIhwf3/P58+fGYaBz58/77kTTAOjnMH1eqLm2cfav6O9vT52cTwcHHubvI9epy82SylcXl6O/fBxd2Y4SkC8v15nXPTERa33P7qROHiFnX2YA+HIhEev8yj3eEl5MZjNNRn4S0YSigpJE7WFDyFizKcKFuxlACY7oBQZvUcR2651ptW/chLFsoU5I4saKJYIjExl2SK9qDU1thdjW6VaOlJRam1AW63tWcYq9mBOAmqTSeQZdFfCfK683sJqraTznvsfC6uV0HXC8qqj6xJUuPl+S/4gyCbzV38j9K1C6YRuadDMgRJ1TG+A8XyZbiHITEEKmjJlI6xSQdaCbpWyBhVhVkAolI1w/wFqzSxmlcVXJlVI/iDPwvKrwn/1Tz3/6T/2/P4/3dJ1/RjUVUk83qy5mAtpMUO6Ytv31e5bQUm9eb5qy9ylAt1CuLwQPp7Dw43JQ5Y5UebGjG+2IA9QrjCHh2oWbAWzLsvYakaKgtYGlJUq2SQnRUdPV3q7t0nt9Zp2C6Isia63YL+6UcrW7MMszkzpUoLO7v+2Cn0DxxZYyJjNTXZTiNKSeUhDuMnlCamOADug6i/emsu/YJx18C9EZyf89+mXSDz/GBA4BPqmrOZzAdUxdvTYccfKU6BlyrRO2/0c4PZSYHSozkNg9RCTO2WQj7Xr2Dl/Tpm25dBCIQKBeH3VXQR7nEvx+MikRtDjrJiDBmcf45yNmdFiLvpji6+/JMP9/1e5vr7eGzcvbrbvEfCR0XJQ43ICl2s44L+6umI2m3
```

Comment:
I think it's more educational to not abstract away how the layers are frozen. After all, isn't that one of the purposes of the tutorial to demonstrate fine-tuning?

Reply:
I agree that making it a little more explicit will be helpful.